tests/_utils/kafka.py

```diff
@@ -0,0 +1,152 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import subprocess
+import time
+import typing
+from collections import namedtuple
+from functools import partial
+
+import pytest
+
+if (typing.TYPE_CHECKING):
+    from kafka import KafkaConsumer
+
+# Amount of seconds we will wait for the Kafka consumer to be assigned partitions
+PARTITION_ASSIGNMENT_TIMEOUT = 15
+
+# actual topic names not important, but we will need two of them.
+KafkaTopics = namedtuple('KafkaTopics', ['input_topic', 'output_topic'])
+
+# pylint: disable=invalid-name
+_kafka_consumer = None
+kafka_server = None
+zookeeper_proc = None
+# pylint: enable=invalid-name
```

Comment:
what's with the pylint directives? aside from the `_kafka_consumer`, these look pretty normal?

Reply:
Pylint believes these to be constants and wants an upper-case name.

Reply:
weird!